File: c/Ex15.c
#include <stdio.h>
int main(int argc, char *argv[])
{
  float peso, altura, imc;
  
  printf("Digite o seu peso em Kg: ");
  scanf("%f", &peso);
  printf("Digite a sua altura em M: ");
  scanf("%f", &altura);
  imc = peso / (altura*altura);
  if (imc < 20)
  {
      printf("Abaixo do peso!");
  }
  else if(imc < 25)
  {
      printf("Peso ideal");
  }
  else
  {
      printf("Acima do peso!");
  }
 
  return 0;
}

File: python/Ex15.py
peso = float(input("Digite o peso: "))
altura = float(input("Digite o altura: "))
IMC = peso / altura**2
if(IMC < 20):
    print("Pessoa esta abaixo do peso")
else:
    if(IMC < 25):
        print("Pessoa tem peso normal")
    else:
        if(IMC < 30):
            print("Pessoa esta acima do peso")
        else:
            print("Pessoa esta Obesa")
